docs/concepts/protocol/02-stream-types.mdx

```diff
@@ -143,3 +145,114 @@ accumulate, so if the recipient doesn't withdraw them, they will be able to with
 their tokens.
+
+### Monthly Unlocks
+
+Monthly Unlocks is a variation of the Unlock in Steps model, with the distinction of having monthly intervals. This
+model is particularly suited for structured yet flexible token distribution plans such as distributing a set salary
+across a year.
+
+<FunctionPlot
+  options={{
+    xAxisLabel: "x - Months",
+    xDomainLimit: 12,
+    yDomainLimit: 120,
+    data: [
+      {
+        fnType: "points",
+        graphType: "polyline",
+        color: "#f77423",
+        points: Array.from({ length: 24 }, (_, i) => [
+          1 + Math.floor(i / 2),
+          10 * Math.floor(i / 2) + (i % 2 === 1 ? 10 : 0),
+        ]),
+      },
+    ],
+  }}
+/>
+
+Each month, on a preset day, a specific amount of tokens becomes unlocked and available for withdrawal. Like the Unlock
+in Steps model, unwithdrawn tokens will carry over to the next period, providing flexibility and control to the
+recipient. This model is ideal for employers who wish to set up advanced payment schedules for their employees, offering
+them access to funds on a predictable, monthly basis.
+
+### Timelock
+
+The Timelock model enables the locking of a certain amount of tokens for a specified period. This model is designed to
+foster longer-term investment and holding strategies by deferring availability.
+
+<FunctionPlot
+  options={{
+    data: [
+      {
+        points: [
+          [0.01, 0.01],
+          [90, 0],
+          [90, 99.7],
+          [99.99, 99.99],
+        ],
+        fnType: "points",
+        graphType: "polyline",
+        color: "#f77423",
+      },
+    ],
+  }}
+/>
+
+Once the set period elapses, the full amount becomes accessible to the recipient. This setup is particularly
+advantageous for projects seeking to stabilize token pricing and minimize market volatility, encouraging investors to
+maintain their stake over a more extended period.
+
+### Unlock Linear
+
+The Unlock Linear model combines an initial immediate release of tokens with a steady, linear payout over time. This
+model is ideal for employment contracts that include a signing bonus along with a regular payout schedule.
+
+<FunctionPlot
+  options={{
+    data: [
+      { fn: "25", range: [0, 0.15], color: "#f77423" },
+      { fn: "(x-0.15)*0.75015 + 25", range: [0.15, 100], color: "#f77423" },
+      {
+        points: [
+          [0.15, 0],
+          [0.15, 25],
+        ],
+        fnType: "points",
+        graphType: "polyline",
+        color: "#f77423",
+      },
+    ],
+  }}
+/>
+
+At the beginning of the stream, a fixed amount of tokens is instantly available to the recipient — this is your "signing
+bonus". Following this, the remaining tokens begin to stream continuously at a consistent rate until the end of the
+contract term — this is your "salary".
+
+### Unlock Cliff
+
+The Unlock Cliff model introduces an unique streaming pattern that combines an initial unlock, a lock phase and then a
+linear unlock phase.
```

Comment:
How about we add a use case here as well (like all the previous ones)?

This shape is useful for companies who want to distribute tokens to their investors using a cliff followed by linear vesting but also want to unlock some liquidity at the beginning to be able to allow investors to bootstrap AMM pools.